test/jdk/java/text/Format/MessageFormat/CompactSubFormats.java

```diff
@@ -42,11 +42,15 @@
 
 public class CompactSubFormats {
 
+    // This test expects an en_US locale, as this locale provides distinct instances
+    // for different styles.
+    private static final Locale loc = Locale.of("en", "US");
```

Comment:
No need to define this field. You can directly use the `Locale.US` constant in the constructor.